File: remd.py
def main():
    l=[1,2,2,3,1,3]
    l=list(set(l))
    # l1=0

    # for ch in l:
    #     l1+=1
    # for i in range(l1-1):
    #     for j in range(i+1,l1):
    #         if l[i]==l[j] and l[i]!=' ':
    #             l[j]=' '
    # count=0
    # for k in range(l1):
    #     if l[k]!=' ':
    #         l[count]=l[k]
    #         count+=1
    # l=l[:count]
    print(l)

if __name__=="__main__":
    main()

File: remd.c
#include<stdio.h>

int main()
{

    int a[]={1,1,2,3,3,4};
    int lena=sizeof(a)/sizeof(a[0]);
    int count=0;

    for(int i=0;i<lena;i++)
    {

        int j=i+1;

            if(a[i]!=a[j]){

                a[count++]=a[i];
            }
            if(a[i]==a[count] && i==lena-1){

                i=lena-1;
            }
    }
    // printf("%d",a[lena]);
    for(int i=0;i<count;i++){

        printf("%d",a[i]);
    }
}